src/components/Home/career-transition-section/career-transition-content/meet-soujunior.tsx

```diff
@@ -0,0 +1,40 @@
+export const MeetSouJunior: React.FC = () => {
+    return (
+        <div
+            className="flex flex-col items-center gap-4 text-center
+                        md:gap-[1.125rem]
+                        lg:items-start"
+        >
+            <h2
+                className="text-blue-950 text-xl font-semibold font-['Radio_Canada'] leading-normal
+                            md:text-4xl md:leading-10
+                            lg:leading-[3rem]"
+            >
+                Conheça a SouJunior
+            </h2>
+
+            <div
+                className="text-center justify-center
+                        lg:text-start"
+            >
+                <p
+                    className="text-blue-900 text-sm font-normal font-['Radio_Canada'] leading-tight
+                            md:text-base md:leading-snug
+                            "
+                >
+                    A {''}
+                    <span className="font-semibold">SouJunior {''}</span>
```

Comment:
o Strong é mais indicado para esses casos

```suggestion
                    <strong className="font-semibold">SouJunior {''}</span>
```